src/index.ts

```diff
@@ -264,10 +268,16 @@ async function wranglerCommands() {
 				env: process.env,
 			},
 		);
+
 		info(result.stdout);
 		return result;
 	});
-	await Promise.all(arrPromises);
+
+	await Promise.all(arrPromises).catch((result) => {
+		error(`🚨 Command failed`);
+		setFailed(result.stderr);
```

Comment:
While this is an improvement, this still will only output error messages if wrangler exits with a non-zero code (which it should, but doesn't always adhere to currently). I think it would be better to move the `result.stderr` logging to below the `info()` call in the block above (conditionally; we shouldn't log anything if `stderr` is empty). It's still good to call `setFailed()` at this level here, but with `🚨 Command failed` rather than the stderr message.

We also need to be sure to handle this for the `defaultCommandBehavior()`. Even better would be to consolidate these codepaths by instead setting a fallback value for `commands`.